mixer/adapter/list/list.go

```diff
@@ -79,7 +80,31 @@ func (h *handler) HandleListEntry(_ context.Context, entry *listentry.Instance)
 		return adapter.CheckResult{}, err
 	}
 
-	found, err := l.checkList(entry.Value)
+	var value string
+	isString := true
+
+	switch entryVal := entry.Value.(type) {
+	case *v1beta1.Value:
+		strVal := entryVal.GetStringValue()
+		ipVal := entryVal.GetIpAddressValue()
+		if strVal != "" {
+			value = strVal
+		} else if ipVal != nil {
+			// IP_ADDRESS comes in as byte array.
+			value = string(ipVal.Value[:])
```

Comment:
What I mean is to convert an IP address into a full string (192.168.10.123), as opposed to this kind of conversion. If you convert to a fully expanded IP address string, then checkList can remain unchanged,

Reply:
Ahh got it... please see if this looks good now..